### ISSUE TITLE
'int google::protobuf::MessageLite::ByteSize() const' is deprecated

### ISSUE THREAD

**Issue by z3ntu** [MirServer/mir#1127]
```
/home/pmos/build/src/mir-1.5.0/src/client/display_configuration.h:48:33: warning: 'int google::protobuf::MessageLite::ByteSize() const' is deprecated: Please use ByteSizeLong() instead [-Wdeprecated-declarations]
   48 |         return wrapped.ByteSize();
      |                                 ^
```

**Comment by AlanGriffiths** [comment]
`ByteSizeLong()` isn't available in 3.0 (as seen in Ubuntu 18.04LTS) nor 2.6 (Ubuntu 16.04LTS), so we can't simply switch to using it. Looks like it was added in 3.1.

@z3ntu, feel like PRing a fix?

PS I'm looking forward to the day we drop protobuf!

**Comment by z3ntu** [comment]
See #1131
